openapi/components/schemas/UserSubscription.yaml

```diff
@@ -1,64 +1,63 @@
 type: object
 description: ''
-example:
-  id: vrchatplus-yearly
-  transactionId: txn_63231aa6-8b7a-49ff-a106-d5678f8318b1
-  store: Steam
-  steamItemId: '5000'
-  amount: 9999
-  description: VRChat Plus (Yearly)
-  period: year
-  tier: 5
-  active: true
-  status: active
-  expires: '2022-03-04T00:00:00.000Z'
-  created_at: '2020-12-04T13:09:54.344Z'
-  updated_at: '2020-12-10T19:53:45.362Z'
-  licenseGroups:
-    - lgrp_608513da-b213-4e15-80af-bd88c27f0979
-  isGift: false
 properties:
   id:
     type: string
     minLength: 1
+    example: 'vrchatplus-yearly'
   transactionId:
     $ref: ./TransactionID.yaml
   store:
     type: string
     minLength: 1
     description: Which "Store" it came from. Right now only Stores are "Steam" and "Admin".
+    example: 'Steam'
   steamItemId:
     type: string
     minLength: 1
+    example: '5000'
   amount:
     type: number
+    example: 9999
   description:
     type: string
+    example: 'VRChat Plus (Yearly)'
   period:
     $ref: ./SubscriptionPeriod.yaml
   tier:
     type: number
+    example: 5
   active:
     type: boolean
     default: true
+    example: true
   status:
     $ref: ./TransactionStatus.yaml
+  starts:
+    type: string
+    minLength: 1
```

Comment:
`minLength: 1` violates this.
https://github.com/vrchatapi/specification-test/blob/main/data/requests/economy/getcurrentsubscriptions.md?plain=1#L55

Reply:
I added `minLength: 1`, because an empty string is not a `date-time`

Reply:
Wouldn't this cause validation to fail in this case?

Reply:
Yes, that would seem the case.

> Note that an empty string "" is a valid string unless minLength or [pattern](https://swagger.io/docs/specification/data-models/data-types/#pattern) is specified

from https://swagger.io/docs/specification/data-models/data-types/

What should we do in this case?

Reply:
We should just remove the constraint.